mhkit/tests/loads/test_loads.py

```diff
@@ -180,6 +180,20 @@ def test_mler_export_time_series(self):
 
         assert_frame_equal(self.mler_ts, mler_ts, atol=0.0001)
 
+    def test_return_year_value(self):
+        dist = stats.norm
+        return_year = 50
+        short_term_period = 1
+
+        val = loads.extreme.return_year_value(dist.ppf, return_year, short_term_period)
+
+        # This is a little hard to test as there is not an indepedent means by
+        # which we can do this calculation. The value below has been taken from
+        # an initial calculation and will indicate if the return_year_value is
+        # changed unexpectedly.
```

Comment:
I think this is acceptable right now, I can think of a couple other tests that do something similar. @ssolson or @cmichelenstrofer thoughts on alternative known values?

Can you remove the comment lines, those will be recorded in this thread.

Reply:
This is a good enough test.

Reply:
Thanks both for confirming.